.changeset/shy-moons-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/vue': minor
+---
+
+New useInterpret & useActor composables + ES Module exports for Vue
```

Comment:
We are not bound to create a single changeset for a package - those are two separate concerns and should be just put into 2 separate changesets

Reply:
Sorry! Haven't used changeset before. Added new changeset now :)